### PR TITLE
Fix field access in ytdata docs.

### PR DESCRIPTION
**Pull request by brittonsmith**
## PR Summary

I stumbled across an error in the [render of the docs](https://yt-project.org/docs/dev/analyzing/saving_data.html#generic-array-data). This should fix it.

## PR Checklist

- [ ] New features are documented, with docstrings and narrative docs
- [ ] Adds a test for any bugs fixed. Adds tests for new features.

### CODE REVIEW

File: doc/source/analyzing/saving_data.rst [hunk 1]
```diff
@@ -227,8 +227,8 @@ selection is not possible, but the data can be accessed through the
    yt.save_as_dataset(ds, "test_data.h5", my_data)
 
    array_ds = yt.load("test_data.h5")
-   print (array_ds.data["region_density"])
-   print (array_ds.data["sphere_density"])
+   print (array_ds.data["data", "region_density"])
+   print (array_ds.data["data", "sphere_density"])
```
Comment (neutrinoceros) [inline doc/source/analyzing/saving_data.rst:231]:
```suggestion
   print(array_ds.data["data", "region_density"])
   print(array_ds.data["data", "sphere_density"])
```

**Review by neutrinoceros — COMMENTED** [review]
While we're at it, might as well align the style with the rest of the docs.
I'll check the docs build when it completes !

**Comment by neutrinoceros** [comment]
my comments are not critical, and I can confirm the fix, so let's get in. Thanks Britton !

**Review by neutrinoceros — APPROVED** [review]

**Comment by brittonsmith** [comment]
Ah, sorry, I had checked out for the day. Feel free to apply that sort of thing to my PRs in the future. Thanks!

**Pull request merged**